React/Modules/RCTStatusBarManager.m

```diff
@@ -108,8 +108,11 @@ - (void)applicationWillChangeStatusBarFrame:(NSNotification *)notification
     RCTLogError(@"RCTStatusBarManager module requires that the \
                 UIViewControllerBasedStatusBarAppearance key in the Info.plist is set to NO");
   } else {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Remove warning.

Reply:
Can you move this change into a separate PR?

Reply:
@cpojer Yes, please see #23625 .